examples/src/operation/bootstrap/bootstrapPool.ts

```diff
@@ -0,0 +1,64 @@
+import { Bootstrap, poolUtils } from "@tinymanorg/tinyman-js-sdk";
+import { getAccount } from "../../util/account";
+import { getAssets } from "../../util/asset";
+import { algodClient } from "../../util/client";
+import signerWithSecretKey from "../../util/initiatorSigner";
+import { SDK_TEST_ARGS } from "../../util/other";
+
+/**
+ * Creates (i.e. "Bootstraps") a pool with an owned asset pair
+ */
+export async function bootstrapPool() {
+  const account = await getAccount();
```

Comment:
here it should just get an address
```suggestion
  const account = await getAccount(accountAddress);
```

and actually we only need the account address. Producing the SK could be the job of the signer function below (to not confuse)

Reply:
`account` includes `addr` and `sk`, we need to pass `sk` to initiator signer.
I think this is not confusing. In these examples we are thinking in node, so user has a single way of signing transactions: using secret key.
So if they want to use this in node, they should have address and secret key.